socorro/collector/wsgi_breakpad_collector.py

```diff
@@ -116,6 +123,18 @@ def POST(self, *args):
         if raw_crash.legacy_processing == IGNORE:
             self.logger.info('%s ignored', crash_id)
             return "Unsupported=1\n"
+        if self.config.collector.reject_crash_on_rahukaalam:
+            # only accept crash if not currently Rahukaalam
+            # http://en.wikipedia.org/wiki/Rahukaalam
+            a = astral.Astral()
+            latitude_mountain_view = '37.3894'
```

Comment:
We'll need to determine the exact latitude and longitude of `us-west-2` in order to properly implement Rahukaalam checking.

Eventually we'll need to do the same for `us-east-1`, which should be easy enough.